content/en/agent/faq/windows-agent-ddagent-user.md

```diff
@@ -3,9 +3,10 @@ title: Datadog Windows Agent User
 kind: faq
 ---
 
-Starting with release `6.11.0`, the Core and APM/Trace components of the Windows Agent run under the `ddagentuser` account, created at install time, instead of running under the `LOCAL_SYSTEM` account, as was the case on prior versions. If enabled, the Live Process component still runs under the `LOCAL_SYSTEM` account.
+Starting with release `6.11.0`, the Core and APM/Trace components of the Windows Agent run under the a dedicated user account, instead of running under the `LOCAL_SYSTEM` account, as was the case on prior versions. If enabled, the Live Process component still runs under the `LOCAL_SYSTEM` account.
 
-The user `ddagentuser` is created at install time for the Datadog Windows Agent. When installed on an Active Directory server, the username and password must be provided to the installer. The new user is a non-privileged user. It gains the following rights during installation:
+The Agent can either use a user-supplied account or create one.
```

Comment:
```suggestion
The Agent installer creates a new account by default (`ddagentuser`) but it can also use a user-supplied account.
```
Let's mention the default early, since it should Just Work (TM) for most users and they can stop reading here.